vendor/vendor.json

```diff
@@ -2796,33 +2794,73 @@
 			"checksumSHA1": "GV2wj5dmLfI8SONK/PApiZO0jKs=",
 			"path": "github.com/shirou/gopsutil",
 			"revision": "fc7e5e7af6052e36e83e5539148015ed2c09d8f9",
-			"revisionTime": "2019-11-27T18:28:36Z",
-			"version": "v2.19.11",
-			"versionExact": "v2.19.11"
+			"revisionTime": "2019-11-27T18:28:36Z"
+		},
+		{
+			"checksumSHA1": "RPWdEuwMMEuNmZNo47yLSeW9MOg=",
+			"path": "github.com/shirou/gopsutil/cpu",
+			"revision": "fc7e5e7af6052e36e83e5539148015ed2c09d8f9",
+			"revisionTime": "2019-11-27T18:28:36Z"
 		},
 		{
 			"checksumSHA1": "h9IgMsg5hwNg5cYMLIx3Ssm+iFg=",
 			"path": "github.com/shirou/gopsutil/disk",
 			"revision": "fc7e5e7af6052e36e83e5539148015ed2c09d8f9",
-			"revisionTime": "2019-11-27T18:28:36Z",
-			"version": "v2.19.11",
-			"versionExact": "v2.19.11"
+			"revisionTime": "2019-11-27T18:28:36Z"
+		},
+		{
+			"checksumSHA1": "obvK/uKg21et4FaUGkbZj02eUm8=",
+			"path": "github.com/shirou/gopsutil/docker",
+			"revision": "fc7e5e7af6052e36e83e5539148015ed2c09d8f9",
+			"revisionTime": "2019-11-27T18:28:36Z"
+		},
+		{
+			"checksumSHA1": "m+trRxaGUONSWWjVIzPM71q4arw=",
+			"path": "github.com/shirou/gopsutil/host",
+			"revision": "fc7e5e7af6052e36e83e5539148015ed2c09d8f9",
+			"revisionTime": "2019-11-27T18:28:36Z"
 		},
 		{
 			"checksumSHA1": "+iFA/hHmdWehBFBa3uAetswSoug=",
 			"path": "github.com/shirou/gopsutil/internal/common",
 			"revision": "fc7e5e7af6052e36e83e5539148015ed2c09d8f9",
-			"revisionTime": "2019-11-27T18:28:36Z",
-			"version": "v2.19.11",
-			"versionExact": "v2.19.11"
+			"revisionTime": "2019-11-27T18:28:36Z"
+		},
+		{
+			"checksumSHA1": "9Ks8cx0QnKNNPud1XYsRCs59dOI=",
+			"path": "github.com/shirou/gopsutil/load",
+			"revision": "fc7e5e7af6052e36e83e5539148015ed2c09d8f9",
+			"revisionTime": "2019-11-27T18:28:36Z"
+		},
+		{
+			"checksumSHA1": "qtcTaosxlOBWPrAE2JT7HuxhLys=",
+			"path": "github.com/shirou/gopsutil/mem",
+			"revision": "fc7e5e7af6052e36e83e5539148015ed2c09d8f9",
+			"revisionTime": "2019-11-27T18:28:36Z"
 		},
 		{
 			"checksumSHA1": "I0kTdVfvfbrboyXnaTeZNmXtT0o=",
 			"path": "github.com/shirou/gopsutil/net",
 			"revision": "fc7e5e7af6052e36e83e5539148015ed2c09d8f9",
-			"revisionTime": "2019-11-27T18:28:36Z",
-			"version": "v2.19.11",
-			"versionExact": "v2.19.11"
```

Comment:
Could we continue using tagged versions when possible?

Reply:
Sure.